moveit_planners/pilz_industrial_motion_planner/package.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>pilz_industrial_motion_planner</name>
+  <version>0.4.10</version>
+  <description>MoveIt! plugin to generate industrial trajectories PTP, LIN, CIRC and sequences thereof.</description>
+
+  <maintainer email="a.gutenkunst@pilz.de">Alexander Gutenkunst</maintainer>
+  <maintainer email="c.henkel@pilz.de">Christian Henkel</maintainer>
+  <maintainer email="i.martini@pilz.de">Immanuel Martini</maintainer>
+  <maintainer email="j.schleicher@pilz.de">Joachim Schleicher</maintainer>
+  <maintainer email="h.slusarek@pilz.de">Hagen Slusarek</maintainer>
+
+  <license>BSD</license>
+
+  <url type="website">https://wiki.ros.org/pilz_trajectory_generation</url>
+  <url type="bugtracker">https://github.com/PilzDE/pilz_industrial_motion/issues</url>
+  <url type="repository">https://github.com/PilzDE/pilz_industrial_motion</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>eigen_conversions</depend>
+  <depend>kdl_conversions</depend>
+  <depend>moveit_ros_planning_interface</depend>
+  <depend>moveit_msgs</depend>
+  <depend>moveit_core</depend>
+  <depend>moveit_ros_planning</depend> <!-- RobotModelLoader -->
+  <depend>moveit_ros_move_group</depend> <!-- move_group capability -->
+  <depend>orocos_kdl</depend>
+  <depend>pilz_extensions</depend>
```

Comment:
Is this package going to be released as a debian regularly? having it as a dependency will require that

Reply:
yes, it is released with all our other packages